packages/main/src/ToolbarItem.ts

```diff
@@ -0,0 +1,66 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import { TemplateFunction } from "@ui5/webcomponents-base/dist/renderer/executeTemplate.js";
+import property from "@ui5/webcomponents-base/dist/decorators/property.js";
+import customElement from "@ui5/webcomponents-base/dist/decorators/customElement.js";
+
+import ToolbarPriority from "./types/ToolbarPriority.js";
+
+@customElement({
+	tag: "ui5-tb-item",
+})
+
+/**
+ * @class
+ * The <code>ui5-tb-item</code> represents an abstract class for items,
+ * used in the <code>ui5-toolbar</code>.
+ *
+ * @constructor
+ * @author SAP SE
+ * @alias ToolbarItem
+ * @extends UI5Element
+ * @public
+ */
+class ToolbarItem extends UI5Element {
+	/**
+	 * Property used to define the access of the item to the overflow Popover. If "Never" option is set,
```

Comment:
Use the established [form of documenting enums](https://github.com/SAP/ui5-webcomponents/blob/main/packages/main/src/Avatar.ts#L132)

```html
 Available options are:
	 * <ul>
	 * <li><code>Never</code></li>
	 * <li><code>Always</code></li>
	 * <li><code>Default</code></li>
	 * </ul>
```